### PR TITLE
Replace -X/-Y in CLI with --cv catboost_clickhouse_sprint_02.02.2019 …

### PR DESCRIPTION
**Pull request by tswr**
Remove -X/-Y from CLI. Replace with --cv Classical:n;k and --cv Inverted:n;k.

Note that docs at https://tech.yandex.com/catboost/doc/dg/concepts/cli-reference_cross-validation-docpage/ should be updated as follows.

```
Execution format

catboost fit -f <file path> [--cv <value>] [--cv-rand <value>] [other parameters]

Options
...
--cv 
Perform cross validation to score the model. Supports two modes. Classical: perform cross validation to score the model by excluding a small fold from the training dataset and using it for testing. Inverted: perform cross validation to score the model by excluding a small fold from the validation dataset and using it for training.

Format for specifying values:

<cvtype>:<n>;<k>
cvtype is either Classical or Inverted.
 n is the identifier of the fold to exclude.
 k is the number of folds to split the input data into.
The inequality n < k must be true.

For example, to exclude the fold indexed 13 from the training dataset and use it for testing, set the value Classical:13;15. To exclude the fold indexed 13 from the validation dataset and use it for training, set the value Inverted:13;15.

The data is randomly shuffled before splitting.

--cv-rand
...
It must be used with --cv <value> parameter.

```

--
I hereby agree to the terms of the CLA available at  https://yandex.ru/legal/cla/?lang=en.

### CODE REVIEW

**Comment by annaveronika** [comment]
@arcadia-devtools Ship it!

**Comment by annaveronika** [comment]
@arcadia-devtools Ship it!

**Comment by annaveronika** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@annaveronika, internal review request created: 701499

**Comment by arcadia-devtools** [comment]
@annaveronika, internal review request created: 701515

**Comment by annaveronika** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@annaveronika, internal review request created: 701529

File: catboost/pytest/test.py [hunk 1]
```diff
@@ -1455,7 +1460,7 @@ def test_cv(is_inverted, boosting_type):
         '-w', '0.03',
         '-T', '4',
         '-m', output_model_path,
-        ('-Y' if is_inverted else '-X'), '2/10',
+        '--cv', format_crossvalidation(is_inverted, 2, 10),
```
[the remark this answers is not in the document]
Comment (tswr) [reply catboost/pytest/test.py]: да, исправил

**Comment by annaveronika** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@annaveronika, internal review request created: 701591

**Pull request merged**